app/views/checkout/payment.scala.html

```diff
@@ -1,16 +1,21 @@
-@(form: Form[model.SubscriptionData])
+@(form: Form[model.SubscriptionData], userIsSignedIn: Boolean)(implicit request: Request[_])
```

Comment:
Well spotted! Turns out that the `request` parameter [is unnecessary](https://www.playframework.com/documentation/2.4.x/api/scala/index.html#play.api.mvc.Call@path():String) here.